README.md

```diff
@@ -1,20 +1,78 @@
-# handling-peer-dependencies
+# How to handle peer dependencies when developing modules 
 
-`yarn` and `npm` doesn't provide tools to install peer dependencies for your development environment. There are an opened issue on yarn repository since [27 October 2016](https://github.com/yarnpkg/yarn/issues/1503).
+## What is a peer dependency and what is the problem with it ?
 
-## Steps
+[Peer dependencies](https://nodejs.org/es/blog/npm/peer-dependencies/#the-solution-peer-dependencies) is a specific kind of dependencies really useful for reusable modules. It lets you ask user to install a dependency your module needs to work. It prevents having multiple version of a same module in user's app node_modules. It also have other benefits like 
```

Comment:
`are a specific kind`

Reply:
Peut-être écrire une liste plutôt que de répéter `it` à chaque début de phrase ?